src/Core/Resolvers/MsSqlQueryBuilder.cs

```diff
@@ -472,8 +472,10 @@ private static string BuildProcedureParameterList(Dictionary<string, object> pro
         /// </summary>
         public string BuildStoredProcedureResultDetailsQuery(string databaseObjectName)
         {
+            // The columns are renamed to match names across different sql implementations.
```

Comment:
nit: only `TYPE_NAME(system_type_id)` is renamed using "as". To add clarity, I suggest that the comment specify that certain columns must not be aliased to maintain compatibility across db types.

Reply:
fixed!